hllplus/hllplus.go

```diff
@@ -197,6 +199,8 @@ func (s *HLL) Downgrade(precision, sparsePrecision uint8) error {
 		return err
 	}
 
+	// TODO: downgrade sparse as well (and don't forget a switch between normal and sparse)
```

Comment:
Is it "downgrade-able"?

Reply:
yes, process is pretty much the same, something for later